src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
Generally it's thought to be dynamic, configurable and extendable, like the enrichment and validation. But I agree with you that it's not a good approach that the user has to get an access token to be able to read some documentation. Still, for me it's not part of the api. Maybe it is some other external documentation and would fit into a document like the Readme. We can also use the wiki here on github. I still have the feeling that a strategy (no matter if it's partition, validation or enrichment) should bring along it's explanation. As an intermediate solution, I'm fine with it.

Anyway, if we remove the `doc` field, only the `name` remains. I'd remove that field as well and just have a string with a name referring to a strategy (as mentioned above).